test/test_aot_autograd.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env pytest
+import functools
+
+import torch
+
+import torchdynamo
+from torchdynamo.optimizations.training import is_aot_autograd_safe_to_run
+from torchdynamo.testing import rand_strided
+
+
+def compiler_safe_fn(gm, example_inputs, is_safe):
+    is_safe[0] = is_aot_autograd_safe_to_run(gm, example_inputs)
```

Comment:
I don't care too much, as its for a test, but a c style out variable isn't the most pythonic thing ever ;) consider returning `gm.forward,  is_aot_autograd_safe_to_run(gm, example_inputs)`